test/types/string/methods/testDedent.good

```diff
@@ -0,0 +1,74 @@
+[0]
+a
+b
+c
+---
+[1]
+
+a
+b
+c
+
+---
+[2]
+
+a
+
+b
+c
+
+---
+[3]
+a
+ b
+ c
+---
+[4]
+ a
+b
+c
+d
+ e
+---
+[5]
+a
+     b
+     c
+---
+[6]
+a
+  b
+		c
+	 d
+ 	e
+
+---
+[7]
+
+    a
+  b
+c
+  d
+    e
+
+---
+[8]
+ a
+b
+c
+---
+[9]
+a
+     b
+     c
+---
+[10]
+a
+    b
+    c
+---
+[11]
+a
```

Comment:
Should this still have a space in front of it because of the ignoreFirst=true?

Reply:
I knew I made some mistakes!